tests/spread/plugins/dotnet/run/task.yaml

```diff
@@ -0,0 +1,25 @@
+summary: Build and run a basic dotnet snap
+
+environment:
+  SNAP_DIR: ../snaps/dotnet-hello
+
+# This plugin does not support core18 and core16 is stable yet
+manual: true
+systems: [ubuntu-16*]
```

Comment:
Should the systems filter be extracted to the spread.yaml like the other suites?